src/sentry/replays/usecases/ingest/issue_creation.py

```diff
@@ -23,6 +23,7 @@ def report_rage_click_issue_with_replay_event(
     url: str,
     node: dict[str, Any],
     replay_event: dict[str, Any],
+    component_name: str | None,
```

Comment:
nit: would prefer if this argument were before replay_event in the parameter order.

Reply:
no prob, can update